src/app/index.test.js

```diff
@@ -88,36 +117,30 @@ describe('generator:app', () => {
   })
   describe('CODE_OF_CONDUCT.md', () => {
     it('contains author email', async () => {
-      await helpers.run(__dirname).withPrompts({
-        pluginName: 'danger-plugin-fun-time',
-        description: 'Danger plugin that tells you to have a fun time',
-        authorName: 'Macklin Underdown',
-        authorEmail: 'email@example.com',
-      })
+      await helpers.run(__dirname).withPrompts(exampleSettings)
       expect(readFile('CODE_OF_CONDUCT.md')).toMatchSnapshot()
     })
   })
   describe('LICENSE.md', () => {
     it('contains the current year and author name', async () => {
       Date.prototype.getUTCFullYear = jest.fn(() => 2017)
-      await helpers.run(__dirname).withPrompts({
-        pluginName: 'danger-plugin-fun-time',
-        description: 'Danger plugin that tells you to have a fun time',
-        authorName: 'Macklin Underdown',
-        authorEmail: 'email@example.com',
-      })
+      await helpers.run(__dirname).withPrompts(exampleSettings)
       expect(readFile('LICENSE.md')).toMatchSnapshot()
     })
   })
   describe('package.json', () => {
-    it('fills package.json with correct information', async () => {
+    it('fills package.json with correct JS information', async () => {
       await helpers.run(__dirname).withPrompts({
-        pluginName: 'danger-plugin-fun-time',
-        description: 'Danger plugin that tells you to have a fun time',
-        authorName: 'Macklin Underdown',
-        authorEmail: 'email@example.com',
-        githubUsername: 'macklinu',
-        keywords: ['fun', 'time'],
+        ...exampleSettings,
+        useTypeScript: false,
+      })
+      const pkg = readFile('package.json')
+      expect(pkg).toMatchSnapshot()
+    })
+    it('fills package.json with correct TS information', async () => {
+      await helpers.run(__dirname).withPrompts({
+        ...exampleSettings,
+        useTypeScript: true,
       })
```

Comment:
these two tests are not quite perfect, they both might be writing to the same file as the `fills package.json with correct JS information` one will correctly show it's JS package if you do `it.only` - but won't right now